client/src/components/FloatingError.js

```diff
@@ -0,0 +1,10 @@
+import { PropTypes } from "prop-types";
```

Comment:
I have an idea. When there is a family of components (like buttons or error), we could group them in a folder (for example components/errors/.*Error.js) :) 
Moreover, it would allow you to move files, and françois loves that.

Reply:
Good idea! I propose a `status/` family, with `PageError.js`, `FloatingError.js` and `Loading.js`. Is this good for you?

Reply:
Great 👍